content/en/user-manual/xr/ar/zappar-integration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Zappar Integration
+layout: usermanual-page.hbs
+position: 2
+---
+
+## Zappar Integration
+
+Zappar offers an [Universal AR SDK with PlayCanvas integration][1], including image, face and instant world tracking features. Their SDK supports Chrome for Android and Safari 11.3+ for iOS, reaching billions of users worldwide.
+
+Zappar provides a [Getting Started with PlayCanvas][2] page, and they also provide a number of [Starter Projects][3] to help new users get started, as well as template projects to get users up and running faster.
+
+## Getting started with Zappar
+
+To get started with Zappar’s Universal AR SDK, first head to their [Zappar Universal AR Project][4], and fork it.
```

Comment:
Should we link to the same project as the one on the new projects page?

https://playcanvas.com/project/797342/overview/instant-tracking--particles

Reply:
Link changed.